prql-compiler/README.md

```diff
@@ -20,6 +20,18 @@ cargo install prql-compiler
 cargo install --path .
 ```
 
+It can be installed via brew! First, install the tap:
+
+```sh
+brew tap prql/homebrew-prql
```

Comment:
This can all just be `brew install prql/prql/prql-compiler`!

Reply:
changed!